blog/2024-04-19-what-is-the-10101-coordinator.md

```diff
@@ -0,0 +1,81 @@
+---
+slug: what is the 10101 coordinator
+title: What is the 10101 coordinator?
+authors: [richard]
+tags: [DLC-Channels]
+---
+
+![](../static//2024-04-19-what-is-the-10101-coordinator/coordinator.png)
+
+10101 aims to bring self-custodial p2p perpetual futures trading to users without sacrificing UX. Centralized exchanges are easy as they only have to keep a book updating all events on their own system. But 10101 is decentralized. Every user is running their own trading service through the 10101 app or dlc connect web app.
```

Comment:
> But 10101 is decentralized

I'm always careful with calling us _decentralized_. Do you think this is correct given that the coordinator is a centralized entity?

Reply:
The current operational model is not decentralized, but the architecture would definitely allow for it. I can change it.

I tried to argue from a point of view that the vision (idea) of 10101 is decentralized as the coordinator can be replaced with any other coordinator service provider - or 10101 nodes could even interact with each other directly.

I think in that sense we are as decentralized as any self-custodial lightning wallet using a LSP.

Reply:
After thinking about it, I think the better term here is `distributed`

Reply:
I like distributed!